### ISSUE TITLE
Single tap cause exception

### ISSUE THREAD

**Issue by CatTail** [gnab/remark#155]
In chrome v38, open dev tool, simulate touch screen.

When using swipe slide switch all right, but a single tap will cause the exception.

`Uncaught TypeError: undefined is not a function`

In code
```javascript
self.events.on("tap", function(endX) {
*    if (endX < self.getContainerWidth() / 2) {
        self.slideshow.gotoPreviousSlide()
    } else {
        self.slideshow.gotoNextSlide()
    }
})
```

The line prefix with `*` is causing the exception.

**Comment by gnab** [comment]
Sorry for the late reply. The fix for this issue will be included in the next relase.

**Comment by gnab** [comment]
This has now been released in version 0.12.
